fluent-tests/Initialize-Tests.ps1

```diff
@@ -130,7 +130,7 @@ $job = @(
     "--version=$AUTOREST_CORE_VERSION $FLUENTLITE_ARGUMENTS --regenerate-pom=false --input-file=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/196886564583ff59186bd0ef44d923120aaf3f78/specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/stable/2023-06-15/NetworkFabrics.json --java.namespace=com.azure.mgmtlitetest.managednetworkfabric",
 
     # polymorphic models in different packages
-    "--version=$AUTOREST_CORE_VERSION $FLUENTLITE_ARGUMENTS --regenerate-pom=false --input-file=https://github.com/Azure/azure-rest-api-specs/blob/64337ca106a20055b389f9652ea8cf942aa94252/specification/consumption/resource-manager/Microsoft.Consumption/stable/2023-11-01/consumption.json --java.namespace=com.azure.mgmtlitetest.consumption"
+    "--version=$AUTOREST_CORE_VERSION $FLUENTLITE_ARGUMENTS --regenerate-pom=false --input-file=https://github.com/Azure/azure-rest-api-specs/blob/64337ca106a20055b389f9652ea8cf942aa94252/specification/consumption/resource-manager/Microsoft.Consumption/stable/2023-11-01/consumption.json --java.namespace=com.azure.mgmtlitetest.consumption --generate-tests=false"
```

Comment:
Model mock test seems having trouble identifying the right child class:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=3595400&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=738c417a-4a48-5620-b571-4f9e77e0bfd2&l=3290
```java
@org.junit.jupiter.api.Test
    public void testDeserialize() throws Exception {
        ModernReservationRecommendation model = BinaryData.fromString(
            "{\"kind\":\"legacy\",\"properties\":{\"scope\":\"uewmrswnjlxuzrhw\",\"location\":\"sxjb\",\"lookBackPeriod\":1129684980,\"instanceFlexibilityRatio\":40.48269,\"instanceFlexibilityGroup\":\"dohzjq\",\"normalizedSize\":\"ucoig\",\"recommendedQuantityNormalized\":38.62258,\"meterId\":\"2668484c-c47e-440c-9b15-69e796f1b9db\",\"term\":\"nwfepbnwg\",\"costWithNoReservedInstances\":{\"currency\":\"jgcgbjbgdlfgtdys\"},\"resourceType\":\"flq\",\"totalCostWithReservedInstances\":{\"currency\":\"q\"},\"netSavings\":{\"currency\":\"jrwdkqz\"},\"firstUsageDate\":\"2021-01-23T02:59:42Z\",\"skuProperties\":[{\"name\":\"iunjxdf\",\"value\":\"ntkwcegy\"},{\"name\":\"lbnseqac\",\"value\":\"vpilg\"},{\"name\":\"oq\",\"value\":\"gmditgueiookjbs\"},{\"name\":\"rtdtpdelq\",\"value\":\"slmot\"}],\"skuName\":\"bnfxofvc\"},\"etag\":\"irazftxejwabmd\",\"tags\":{\"kyqltqsrogt\":\"mvcopexcmjurbuhh\",\"ysidfvclgl\":\"wkffdjkt\"},\"location\":\"fuijtkbus\",\"sku\":\"gsfikayiansha\",\"id\":\"ujtjiqxfz\",\"name\":\"j\",\"type\":\"ttvwkpqh\"}")
            .toObject(ModernReservationRecommendation.class);
}
```
Disabling test for it for now.